.github/workflows/main.yml

```diff
@@ -45,6 +45,46 @@ jobs:
         run: |
           poetry config virtualenvs.in-project true
           poetry install --no-interaction
+
+      - name: Run tests
+        run: poetry run pytest -v
+  lint:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.11"]
+    runs-on: ubuntu-18.04
```

Comment:
Probably want to update to a newer ubuntu release as this is EOL by April 30th, 2023

https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/

Reply:
Mind if I make a separate issue for that?

Reply:
https://github.com/uSpike/rpcx/issues/70

Reply:
Sure that works.